Fixed concatenation benchmark + added it to the microbenchmarking runs

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#31587 Fixed concatenation benchmark + added it to the microbenchmarking runs**

Differential Revision: [D19221813](https://our.internmc.facebook.com/intern/diff/D19221813)

Commit:
Fixed concatenation benchmark + added it to the microbenchmarking runs

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 5f2285cf:


* **1/1** broken upstream at merge base 68e51723 (see [grid view](https://dr.pytorch.org/master-timeline.html?count=1&sha1=68e517238291d1149a89d1f4e9fc90b560804d00&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35))
  <details>
  <summary>
  You may want to rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  
* **0/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=5f2285cfb0b6ce0c8233519c828dc2f4135c6746).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably caused by upstream breakages:
* [pytorch_linux_xenial_py3_clang5_asan_test](https://dr.pytorch.org/build-details.html?build_id=78592634)



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

Commit:
Update on "Fixed concatenation benchmark + added it to the microbenchmarking runs"


Differential Revision: [D19221813](https://our.internmc.facebook.com/intern/diff/D19221813)
